modules/Kconfig.mcuboot

```diff
@@ -208,4 +208,10 @@ config BOOT_IMAGE_ACCESS_HOOKS
 	  It is up to the application project to add source file which
 	  implements hooks to the build.
 
+config MCUBOOT_BOOTUTIL_LIB_FOR_DIRECT_XIP
+	bool "DirectXIP support in bootutil"
```

Comment:
Shouldn't this depend on the mcuboot configured mode Kconfig (the zephyr app one, not the mcuboot one) being XIP?

Reply:
Made it hidden option selected by the MCUboot mode Kconfig.